lib/metrics/services.js

```diff
@@ -2,13 +2,14 @@ module.exports = {
 	'acast': /^https:\/\/media\.acast\.com/,
 	'acast-thumbnails': /^https:\/\/thumborcdn\.acast\.com/,
 	'access': /^https:\/\/access\.ft\.com/,
-	'access-licence-svc': /^https:\/\/(beta-)?api\.ft\.com\/licences\//,
+	'access-licence-svc': /^https:\/\/(beta-)?api\.ft\.com\/licences\/?,
```

Comment:
this is missing the trailing `/` to close the regex:

```suggestion
	'access-licence-svc': /^https:\/\/(beta-)?api\.ft\.com\/licences\/?/,
```